virtcontainers/clh.go

```diff
@@ -410,6 +412,38 @@ func (clh *cloudHypervisor) getThreadIDs() (vcpuThreadIDs, error) {
 	return vcpuInfo, nil
 }
 
+func (clh *cloudHypervisor) hotplugBlockDevice(drive *config.BlockDrive) error {
+	if clh.config.BlockDeviceDriver != config.VirtioBlock {
+		return fmt.Errorf("incorrect hypervisor configuration on 'block_device_driver':"+
+			" using '%v' but only support 'virtio-blk'", clh.config.BlockDeviceDriver)
```

Comment:
Nit: I'd consider using a `%q` and `config.VirtioBlock` to avoid hard-coding "virtio-blk" in the returned error.